### ISSUE TITLE
Review LOI High Risk

### ISSUE THREAD

**Issue by smiths** [smiths/TCTvMDE#7]
The first draft of the "high risk" page for the LOI is now complete (https://github.com/smiths/TCTvMDE/blob/master/LOI/HighRisk.tex).  Please review.  Feel free to make changes directly in the tex source, or to suggest changes as comments in the discussion below.  When you are done your review, please check your name off on the following list:

- [ ] @JacquesCarette 
- [ ] @richpaige 
- [x] @ZMotamed 

The instructions for this one page document are given below.

Describe the high-risk nature of the project. Provide a rationale for why it should be considered high-risk.  The high risk evaluation criteria are:
- proposing unique directions;
- defying current research paradigms;
- enhancing our understanding of complex and challenging issues.
- novel interdisciplinary approach
- development or adaptation of methods and techniques

I have chosen to cover each criteria in a separate paragraph.

**Comment by ZMotamed** [comment]
Done.

**Comment by smiths** [comment]
I'm reopening this issue, since @JacquesCarette and @richpaige have not yet reviewed the document.

**Comment by smiths** [comment]
Thank you for your comments @ZMotamed.  I added BibTeX entries for the references you added in 95628fb.  I also corrected a few minor typos.  

@ZMotamed, in the following sentence, should "expectations" be "exceptions"?

"Currently, clinical decisions are chiefly made based on the anatomy alone with some
expectations."

**Comment by ZMotamed** [comment]
No, that is "expectations". You can remove it if the word is confusing. 
"Currently, clinical decisions are chiefly made based on the anatomy alone"

**Comment by smiths** [comment]
I've removed "expectations."  You are right that I found it confusing.  😄
